libraries/tock-rt0/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+//! Tock generic initial runtime (`rt0`) helpers
+
+#![no_std]
+
+/// Initializes the `static data`, by copying it into memory (RAM) from
+/// non-volatile memory (Flash)
+// Relocate data segment.
+// Assumes data starts right after text segment as specified by the linker
+pub unsafe fn init_data(mut edata: *mut u32, mut sdata: *mut u32, sdata_end: *mut u32) {
```

Comment:
Seems odd to have `edata` as the argument here. Are these really `start_data_flash`, `start_data_ram`, `end_data_ram`?

Reply:
Right, so let's just rename them so that future developers don't have to back calculate that mapping.